mtenn/tests/test_model_config.py

```diff
@@ -65,3 +66,58 @@ def test_random_seed_schnet():
         for p1, p2 in zip(set_model1.parameters(), set_model2.parameters())
     ]
     assert sum(set_equal) == len(set_equal)
+
+@pytest.mark.skipif(not HAS_VISNET, reason="requires VisNet from nightly PyG")
+def test_random_seed_visnet():
+    rand_config = ViSNetModelConfig()
+    set_config = ViSNetModelConfig(rand_seed=10)
+
+    rand_model1 = rand_config.build()
+    rand_model2 = rand_config.build()
+    set_model1 = set_config.build()
+    set_model2 = set_config.build()
+
+    rand_equal = [
+        (p1 == p2).all()
+        for p1, p2 in zip(rand_model1.parameters(), rand_model2.parameters())
+    ]
+    assert sum(rand_equal) < len(rand_equal)
+
+    set_equal = [
+        (p1 == p2).all()
+        for p1, p2 in zip(set_model1.parameters(), set_model2.parameters())
+    ]
+    assert sum(set_equal) == len(set_equal)
+
+@pytest.mark.skipif(not HAS_VISNET, reason="requires VisNet from nightly PyG")
+def test_visnet_from_pyg():
+    from torch_geometric.nn.models import ViSNet as PyVisNet
+    from mtenn.conversion_utils.visnet import ViSNet
+    kwargs={
+        'lmax': 1,
+        'vecnorm_type': None,
+        'trainable_vecnorm': False,
+        'num_heads': 8,
+        'num_layers': 6,
+        'hidden_channels': 128,
+        'num_rbf': 32,
+        'trainable_rbf': False,
+        'max_z': 100,
+        'cutoff': 5.0,
+        'max_num_neighbors': 32,
+        'vertex': False,
+        'reduce_op': "sum",
+        'mean': 0.0,
+        'std': 1.0,
+        'derivative': False,
+        'atomref': None,
+    }
+
+    pyg_model = PyVisNet(**kwargs)
+    visnet_model = ViSNet(pyg_model)
+
+    rand_equal = [
```

Comment:
should probably rename this just to something a little more relevant (eg `equal_params` or something)